lib/variable.js

```diff
@@ -111,6 +111,11 @@ module.exports = function getVariableAccessors (container, varName, defValue, ex
         throw new EnvVarError(`"${varName}" is a required variable, but it was not set`)
       }
 
+      const value = typeof defValue !== 'undefined' ? defValue : container[varName]
```

Comment:
@xuo I think this should be `typeof  container[varName] === 'undefined' ? defValue : container[varName]` since we always want to use the value defined in the environment and only use the default as a fallback?

Reply:
yeah, that definitely makes much more sense :-)